### PR TITLE
WST Panel with list of translators

### PR DESCRIPTION
**Pull request by danieljames-dj**

### CODE REVIEW

File: WcaOnRails/app/controllers/translations_controller.rb [hunk 1]
```diff
@@ -87,6 +87,30 @@ def update
   }.freeze
   # rubocop:enable Style/NumericLiterals
 
+  def self.translators_to_roles
```
Comment (gregorbg) [inline WcaOnRails/app/controllers/translations_controller.rb:90]: This should be feasible with `map` and `flat_map`:

```ruby
return VERIFIED_TRANSLATORS_BY_LOCALE.flat_map do |locale, user_ids|
  user_ids.map do |user_id|
    {
      ...role data...
    }
  end
end
```

File: WcaOnRails/app/controllers/api/v0/user_roles_controller.rb [hunk 2]
```diff
@@ -170,13 +170,21 @@ def index_for_group_type
...
 
     # Filter the list based on the permissions of the logged in user.
     roles = filter_roles_for_logged_in_user(roles)
 
     # Filter the list based on the other parameters.
-    roles = filter_roles_for_parameters(roles: roles, status: params[:status], is_active: params[:isActive])
+    roles = filter_roles_for_parameters(
+      roles: roles,
+      status: params[:status],
+      is_active: params.key?(:isActive) ? ActiveRecord::Type::Boolean.new.cast(params.require(:isActive)) : nil,
```
Comment (gregorbg) [inline WcaOnRails/app/controllers/api/v0/user_roles_controller.rb:186]: Consider outsourcing this into a separate line of code (or two)

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: WcaOnRails/app/controllers/translations_controller.rb (hunk 1 above)
Comment (danieljames-dj) [reply WcaOnRails/app/controllers/translations_controller.rb:90]: Done, this is nice.

File: WcaOnRails/app/controllers/api/v0/user_roles_controller.rb (hunk 2 above)
Comment (danieljames-dj) [reply WcaOnRails/app/controllers/api/v0/user_roles_controller.rb]: Done

**Review by gregorbg — APPROVED** [review]

**Pull request merged**